automated install/basic-install.sh

```diff
@@ -305,13 +305,15 @@ elif command -v rpm &> /dev/null; then
         fi
     fi
     else
-        # If not a supported version of Fedora or CentOS,
-        echo -e "  ${CROSS} Unsupported RPM based distribution"
-        # exit the installer
-        exit
+        # Warn user of unsupported version of Fedora or CentOS
+        if ! whiptail --defaultno --title "Unsupported RPM based distribution" --yesno "Would you like to continue installation on an unsupported RPM based distribution?\\n\\nPlease ensure the following packages have been installed manually:\\n\\n- lighttpd\\n- lighttpd-fastcgi\\n- PHP version 7+" ${r} ${c}; then
+            exit # exit the installer
+        else
+            : # continue with unsupported RPM distribution
+        fi
     fi
 
-# If neither apt-get or rmp/dnf are found
+# If neither apt-get or yum/dnf are found
```

Comment:
The conditionals check for `rpm` and `dnf`, but don't explicitly check for `yum`. Does this comment still make sense then?

Reply:
I would say it makes more sense as it is now comparing _apple to apples_.
`dpkg` is to Debian as `rpm` is to Fedora.  
`apt-get` is to Debian as `yum` is to CentOS and `dnf` is to Fedora.

I just fixed a typo, we could re-word the whole thing imo.

Reply:
It looks good, no change needed.